### DISCUSSION TITLE
Circular style QR code

### DISCUSSION THREAD

**Discussion by Bob-GGB** [dagronf/QRCode#52] [Ideas]
I really hope the author would consider this style of QR code！

https://github.com/user-attachments/assets/c5baa07e-23d6-4a69-8bcd-822693ad2d32

**Comment by dagronf** [comment]
This circular barcode is just a standard square barcode with some frame adornments (the additional dots and the border bars). 

![image](https://github.com/user-attachments/assets/fb4565ff-3fd5-4884-8ac3-a53a37abc942)

I'm debating as to whether this is outside the scope for this library, as there's infinite ways a potential frame can look depending on the client need. What design works for one client isn't useful for another.

Frames can be added after generating the qrcode by the caller. Maybe there's a way to extend the qr 'dots' outside the qr code so that the qr code style matches - I dunno.
